walkthroughs/howto-ingress-gateway/infrastructure/ecs-service.yaml

```diff
@@ -598,7 +598,7 @@ Resources:
             - Name: CW_CONFIG_CONTENT
               Value:
                 Fn::Sub:
-                  - '{ \"metrics\": { \"namespace\":\"${MetricNamespace}\", \"metrics_collected\": { \"statsd\": { \"metrics_aggregation_interval\": 0}}}}'
+                  - "{ \"metrics\": { \"namespace\":\"${MetricNamespace}\", \"metrics_collected\": { \"statsd\": { \"metrics_aggregation_interval\": 0}}}}"
```

Comment:
Thanks for fixing this. I just discovered yesterday that there's an issue with the cw-agent configuration and was planning to work on this.